procedures/foreman/branch.md.erb

```diff
@@ -20,6 +20,11 @@
 * [ ] Add version <%= release %> to jobs in axes and/or combination filters, remove old ones (keep three)
 * [ ] Clone Debian nightly repos to <%= release %> using [copy/freight instructions](https://projects.theforeman.org/projects/foreman/wiki/Debian_Packaging#Project-sources)
 * [ ] Edit [foreman-packaging's PR template](https://github.com/theforeman/foreman-packaging/blob/master/.github/PULL_REQUEST_TEMPLATE.md) to add <%= release %> and remove the old release
+* [ ] Add foreman-client
+  * [ ] Add client tag configuration to tool_belt config
+  * [ ] Clone tags and create build targets
+  * [ ] Create mash scripts and transfer to Koji
+  * [ ] Create release job in foreman-infra, [example](https://github.com/theforeman/foreman-infra/blob/master/puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/client/1.20.groovy)
```

Comment:
And don't forget the the actual job definition :) Perhaps that's a better example since it links to the required files. Both also works.